rust/chains/hyperlane-ethereum/src/mailbox.rs

```diff
@@ -437,7 +437,7 @@ mod test {
             provider.clone(),
             &ContractLocator {
                 // An Arbitrum Nitro chain
-                domain: &HyperlaneDomain::Known(KnownHyperlaneDomain::ArbitrumGoerli),
+                domain: &HyperlaneDomain::Known(KnownHyperlaneDomain::PlumeTestnet),
```

Comment:
assuming this is fine as they're both on nitro stack

Reply:
cc @tkporter @daniel-savu

Reply:
if the tests pass then it's fine, the `l2_gas_limit` would be `None` if this wasn't an arbitrum nitro chain ([here](https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/dan/prep-queue-labels/rust/chains/hyperlane-ethereum/src/mailbox.rs#L358))